docs/react-native/upgrading.md

```diff
@@ -81,6 +82,47 @@ APIs related to moments should be removed from your code.
 Any place that you were previously instrumenting your app's performance using Moments can now be done using Performance
 Tracing, please refer to [this guide](/react-native/features/traces/) for more information.
 
+## Troubleshooting
+
+The Embrace React Native SDK implements in the shadows the Native version of our product for Android and iOS, neeading at least a sort of minimum versions required to work as expected ([more details here](/react-native/integration/))
+
+## Android
```

Comment:
I think you could go right into the Android issue as a subheading, I moved the link to the required versions to the bottom let me know if that makes sense
```suggestion
### Android
```